### DISCUSSION TITLE
Calling Native by name

### DISCUSSION THREAD

**Discussion by isorokuYam** [scripthookvdotnet/scripthookvdotnet#1409] [Q&A] [answered]
Hello,
I would to use native function "BOOL JUNCTIONS_ACTIVATE_PHASE(VECTOR vJunctionPosition, INT iPhase)", but  there is no hash for it in database (I've got some form the leak, but I heard that they change with every new version). Is it possible to call this by name in SHVDN?

**Comment by kagikn** [comment, marked as answer]
`JUNCTIONS_ACTIVATE_PHASE` is not included in any public final/production builds yet, meaning the function body does not exist in any of them. Native names are not included in the exe, although we have native name enums (in SHVDN and a `natives.h`).
